detox/package.json

```
@@ -14,13 +14,14 @@
     "@types/jest": "29.5.5",
     "@types/tough-cookie": "4.0.3",
     "@types/uuid": "9.0.4",
+    "async": "^3.2.5",
```

Was this use somewhere? If yes, let's use fix version.
```suggestion
    "async": "3.2.5",
```